code/controllers/subsystem/ticker.dm

```diff
@@ -401,7 +401,7 @@ SUBSYSTEM_DEF(ticker)
 			var/acting_captain = !is_captain_job(player_assigned_role)
 			SSjob.promote_to_captain(new_player_living, acting_captain)
 			OnRoundstart(CALLBACK(GLOBAL_PROC, .proc/minor_announce, player_assigned_role.get_captaincy_announcement(new_player_living)))
-		if(ishuman(new_player_living) && CONFIG_GET(flag/roundstart_traits))
+		if(player_assigned_role.job_flags & JOB_ASSIGN_QUIRKS && ishuman(new_player_living) && CONFIG_GET(flag/roundstart_traits))
```

Comment:
```suggestion
		if((player_assigned_role.job_flags & JOB_ASSIGN_QUIRKS) && ishuman(new_player_living) && CONFIG_GET(flag/roundstart_traits))
```
I think this is a mostly stylistic thing but an extra pair of parenthesis around bitflag comparisons for clarity is nice

Reply:
As you said, It's a stylistic choice. It's not an operator like in where it'd work differently when used along other operators without parenthesis separating them.

Reply:
i think its more readable